product-service/helpers/logRequestHelper.js

```diff
@@ -0,0 +1,9 @@
+export const logIncomeRequestEvent = (request, event) => {
+    const { path, queryStringParameters, pathParameters } = event;
+
+    console.log(`request ${request} event: 
+        path: ${path}
+        query parameters: ${JSON.stringify(queryStringParameters)}
+        path Parameters: ${JSON.stringify(pathParameters)}
```

Comment:
I think, it would be convinient to see body of the POST request in the logs.

Reply:
Agree, added